tests/test_cli.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for the CLI."""
+import os
+import pathlib
+import sys
+from unittest import mock
+
+import pytest
+
+from open_alchemy import cli
+from open_alchemy import exceptions
+
+
+@pytest.mark.parametrize(
+    "specfile",
+    [
+        pytest.param("spec.json", id="valid json specfile"),
+        pytest.param("spec.yml", id="valid yml specfile"),
+        pytest.param("spec.yaml", id="valid yaml specfile"),
+    ],
+)
+@pytest.mark.cli
+def test_validate_specfile_valid(tmp_path, specfile):
+    """
+    GIVEN a valid specification file
+    WHEN validate_specfile is called
+    THEN nothing happens
+    """
+    spec = tmp_path / specfile
```

Comment:
I noted this in the `CONTRIBUTING.md` file, please keep a blank line between the code that implements GIVEN, WHEN and THEN. In this case line 28 and 29 to me look like GIVEN, and line 30 is a WHEN and there is no THEN since the expectation is that no error is raised

Reply:
Copy that!